src/cluster.c

```diff
@@ -7361,8 +7361,9 @@ unsigned int delKeysInSlot(unsigned int hashslot) {
         propagateDeletion(&server.db[0], key, server.lazyfree_lazy_server_del);
         signalModifiedKey(NULL, &server.db[0], key);
         moduleNotifyKeyspaceEvent(NOTIFY_GENERIC, "del", key, server.db[0].id);
+        postExecutionUnitOperations();
         decrRefCount(key);
-        propagatePendingCommands();
+        postExecutionUnitOperations();
```

Comment:
@MeirShpilraien isn't redundant?

Reply:
yes, certainly redundant, i wonder how it slipped in....
admittedly, i only did incremental reviews, and didn't do a top to bottom one before merging.
please make a PR to fix.

Reply:
Yes, totally redudent, bad merge from unstable on my side (funny, I believe I also did the PR that caused the conflict and I still miss it)
Will make a PR to remove it.

Reply:
@MeirShpilraien i see sundb already made the PR: #11547

Reply:
Ohh right, greate, thanks.